templates/shared/forms/interactive/container.html

```diff
@@ -0,0 +1,187 @@
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       name="formid"
+                       value="%% formid %%" />
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       name="Consent_to_Processing__c"
+                       value="yes" />
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       name="utm_campaign"
+                       id="utm_campaign"
+                       value="" />
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       name="utm_medium"
+                       id="utm_medium"
+                       value="" />
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       name="utm_source"
+                       id="utm_source"
+                       value="" />
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       name="utm_content"
+                       id="utm_content"
+                       value="" />
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       name="utm_term"
+                       id="utm_term"
+                       value="" />
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       name="GCLID__c"
+                       id="GCLID__c"
+                       value="" />
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       name="Facebook_Click_ID__c"
+                       id="Facebook_Click_ID__c"
+                       value="" />
+                <input type="hidden"
+                       aria-hidden="true"
+                       aria-label="hidden field"
+                       id="preferredLanguage"
+                       name="preferredLanguage"
+                       maxlength="255"
+                       value="" />
+              </div>
+              <div class="pagination p-section--shallow">
+                <hr />
+                <button type="submit"
+                        class="pagination__link--next p-button--positive"
+                        aria-label="Submit">Submit form</button>
+              </div>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="js-pagination js-pagination--2 u-hide">
```

Comment:
I'm not sure this div is needed, given that we route back to the original page and show a thank-you banner now for these instead